### DISCUSSION TITLE
honeybadger-io/js types.d.ts(4,13): error TS1005: '=' expected.

### DISCUSSION THREAD

**Discussion by jerryphe88** [honeybadger-io/honeybadger-js#813] [Q&A]
For honeybadger-io/js version 4.0.3, we got type error
#13 169.8 ERROR in node_modules/@honeybadger-io/js/dist/browser/types/core/types.d.ts(4,13): error TS1005: '=' expected.
#13 169.8 node_modules/@honeybadger-io/js/dist/browser/types/core/types.d.ts(4,40): error TS1005: ';' expected.
#13 169.8 node_modules/@honeybadger-io/js/dist/server/types/core/types.d.ts(4,13): error TS1005: '=' expected.
#13 169.8 node_modules/@honeybadger-io/js/dist/server/types/core/types.d.ts(4,40): error TS1005: ';' expected.
#13 169.8 node_modules/@honeybadger-io/js/dist/server/types/server/aws_lambda.d.ts(2,13): error TS1005: '=' expected.
#13 169.8 node_modules/@honeybadger-io/js/dist/server/types/server/aws_lambda.d.ts(2,49): error TS1005: ';' expected.

We use typescript version 2.9.2

Any idea?

**Comment by subzero10** [comment]
Hey, thanks for creating a discussion! 

The Typescript version is pretty old (latest is 4.7). Have you tried with a newer version?
Or is this not an option for you at this time?

**Reply by atex21** [reply]
hey!
in this case the upgrade of TS is not an option, but it seems that TS version is not the root cause exactly.
In another project, with node, where the TS version is exactly the same honeybadger doesn't give us errors.. I believe the issue is related with types..
Since there are more updates in honeybadger regarding types, do you think can be some types more missing or incompatibility with this project?

Thank you!

**Reply by subzero10** [reply]
Hmm, maybe. But it's difficult to answer without a reproducible example. Is it possible for you to share a repository where this issue can be reproduced so that I can take a look?

Thanks!

**Reply by atex21** [reply]
the repo is private, but I'll try to reproduce the issue in a public repo to show you.
Thank you
